### PR TITLE
MM MetaDB Derived Table Instance notes

### PR DESCRIPTION
**Pull request by jenmawe**
New ported derived table for metaDB for instance_notes

### CODE REVIEW

**Comment by natalyapik** [comment]
Axel, could you please ran this sql again? Mine is saying table folio_inventory.instance__t does not exists. I can see it in the directory. Not sure if I m doing something incorrectly.Thank you.

**Comment by natalyapik** [comment]
all looks good, may be just change INSTANCE to lower case as in other queries: LEFT JOIN folio_inventory.**INSTANCE** as in other queries?

File: sql_metadb/derived_tables/instance_notes.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	it.id AS instance_id,
+	it.hrid AS instance_hrid,
+	jsonb_extract_path_text(instnotes.jsonb, 'note') AS instance_note,
+	jsonb_extract_path_text(instnotes.jsonb, 'staffOnly')::boolean AS staff_only_note,
+	jsonb_extract_path_text(instnotes.jsonb, 'instanceNoteTypeId')::uuid AS instance_note_type_id,
+	intt.name AS instance_note_type_name,
+	instnotes.ordinality AS instance_notes_ordinality
+FROM 
+	folio_inventory.instance__t AS it
+	LEFT JOIN folio_inventory.INSTANCE AS inst ON inst.id::uuid = it.id::uuid
```
Comment (natalyapik) [inline sql_metadb/derived_tables/instance_notes.sql:15]: lower case instance in folio_inventory.INSTANCE

**Review by natalyapik — CHANGES_REQUESTED** [review]

**Review by adoerrer — APPROVED** [review]
Runs great, sql and results looks good, thanks!

All queries:
- [x] Pull request has a complete description
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Review by natalyapik — APPROVED** [review]
yep, looks great

**Pull request merged**